core/src/test/scala/integration/kafka/server/KRaftClusterTest.scala

```diff
@@ -696,7 +696,8 @@ class KRaftClusterTest {
             new AlterConfigOp(new ConfigEntry("max.connections.per.ip", "60"), OpType.SET))))))
         validateConfigs(admin, Map(new ConfigResource(Type.BROKER, "") -> Seq(
           ("log.roll.ms", "1234567"),
-          ("max.connections.per.ip", "60"))), exhaustive = true)
+          ("max.connections.per.ip", "60"),
+          ("min.insync.replicas", "1"))), exhaustive = true)
```

Comment:
Why did we make this change?

Reply:
after https://github.com/brandboat/kafka/commit/ec49a60e4f690f311060045ee1feb20d99837f97, if ELR is enabled, a cluster-level min.insync.replicas will be set to 1, and this PR will enable enable ELR by default in all integration tests that use TestKitNodes.java